spinner/spinner.go

```diff
@@ -60,8 +61,27 @@ func (s *Spinner) Title(title string) *Spinner {
 	return s
 }
 
+// Output set the output for the spinner.
+// Default is STDOUT when [Accessible], STDERR otherwise.
+func (s *Spinner) Output(w io.Writer) *Spinner {
+	s.output = w
+	return s
+}
+
 // Action sets the action of the spinner.
 func (s *Spinner) Action(action func()) *Spinner {
+	s.action = func(context.Context) error {
+		action()
+		return nil
+	}
+	return s
+}
+
+// ActionWithErr sets the action of the spinner.
+//
+// This is just like [Action], but allows the action to use a `context.Context`
```

Comment:
```suggestion
// This is just like [Spinner.Action], but allows the action to use a `context.Context`
```

It's a method on a struct, not a function